DEVELOPMENT.md

```diff
@@ -78,6 +78,18 @@ When enabled by default, this also works with editor plugins like
 
 [vim-fugitive]: https://github.com/tpope/vim-fugitive
 
+### Incremental Build
+
+Bazel is capable of doing incremental builds where it builds only subset of
+files that are impacted by file changes. To use that, just start TensorBoard
+with `ibazel` instead of `bazel`. If you do not have `ibazel` binary on your
+system, you can do use command below.
+
+```sh
+# After running `yarn` in the repo.
+yarn run ibazel run tensorboard -- --logdir [LOG_DIR]
+```
+
```

Comment:
I will opt out of running the test. While the test can be run by `ibazel test //tensorboard/...`, that's often not what you want and our build rules are incapable of doing the right thing in the most cases (karma).

Reply:
Sorry, I meant for running tests in debug mode.
ibazel run tensorboard/components/tf_tensorboard/test:test_web_library

I found that very valuable.

This would be a better place to mention it, actually:
https://github.com/tensorflow/tensorboard/blob/master/DEVELOPMENT.md#debugging-ui-tests-locally

There is the additional complication that it requires other local changes to ensure the port is steady.  Maybe it is possible to add a command line property to enable a steady port?

I can look into this in a future PR so you don't have to deal with it here.

Reply:
Ack.